### ISSUE TITLE
Add Growthcraft

### ISSUE THREAD

**Issue by EnderCreeper15** [Nincraft/NincraftElectricBoogaloo2RevengeoftheFallen#264]
http://www.minecraftforum.net/forums/mapping-and-modding/minecraft-mods/wip-mods/2505072-growthcraft-community-edition-proper-1-7-10

An agriculture mod, originally by Gwafu, which adds a bunch of new crops (and booze). I also contributed a bunch of ideas and textures to it. As you can see in the wiki we're adding lots of new stuff in the coming updates.

The main problem is that Pam's Harvestcraft already adds most of the stuff in Growthcraft. In my opinion Growthcraft is better because it adds more varied content (like the booze production and the different methods of farming) and Pam's is just hundreds of crops that are all basically reskins of one or two things. But it's up to you guys. Also maybe there is a way to have both mods, maybe by disabling some of Pam's content or some of Growthcraft's content so they don't overlap.

Other suggestions:

- Apple trees are kind of overpowered so reduce the apple growth rate, or just change the recipe.

- Disable BoP bamboo forests because GC adds both bamboo and a bamboo forest biome. GC's bamboo has more uses like rafts and decorative blocks.

- Nerf or disable the Fishtrap

**Comment by UndeadZeratul** [comment]
We'd probably also want to try and disable the BoP apple trees

**Comment by EnderCreeper15** [comment]
Yeah, they only generate in the orchard biome right? So you can disable that biome completely, or maybe even find a way to make GC apple trees generate there.

**Comment by UndeadZeratul** [comment]
That's true we could just disable that biome from generating.  If we don't disable BoP's bamboo first, we can always make both bamboo versions compatible
